palabritas-parser/src/main.rs

```diff
@@ -0,0 +1,81 @@
+extern crate pest;
+#[macro_use]
+extern crate pest_derive;
+
+use pest::{Parser, iterators::Pair};
+
+#[derive(Parser)]
+#[grammar = "palabritas.pest"]
+pub struct PalabritasParser;
+
+fn main() {
```

Comment:
I think the idea would be to start from parsing basic elements, and then constructing the more complex ones, all of this being done with tests. right?